test/utils/test_normalization.py

```diff
@@ -0,0 +1,183 @@
+"""Test normalization."""
+
+import numpy as np
+import scipy.sparse.linalg as spl
+from scipy.sparse import csr_matrix, diags
+
+from toponetx.utils.normalization import (
+    _compute_B1_normalized_matrix,
+    _compute_B1T_normalized_matrix,
+    _compute_B2_normalized_matrix,
+    _compute_B2T_normalized_matrix,
+    _compute_D1,
+    _compute_D2,
+    _compute_D3,
+    _compute_D5,
```

Comment:
As far as I see these internal functions are not called directly here and should not be imported.